apple/swift-docc
# fix for replacing the term abstract with summary in diagnostics

Bug/issue #, if applicable: 
rdar://78015631 

## Summary

Diagnostic message that mentions the term 'abstract' is actually referring to 'summary'. The change unifies the terminology in the diagnostic message to use 'summary'. 

## Dependencies 
No known dependencies 

## Testing

1. Create a project in Xcode, it can be a swift project
2. Create .docc folder in that, and a .md file
3. Have an image or a link in a title 
4. Build documentation.
5. Diagnostic to be added for the above and diagnostic message should include the term summary and not abstract. 

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
